homework/transform-containers/transform.cpp

```diff
@@ -0,0 +1,32 @@
+
+#include "transform.hpp"
+#include <algorithm>
+#include <iostream>
+
+void removeDuplicate(std::list<std::string>& lis) {
+    lis.sort();
+    lis.erase(std::unique(lis.begin(), lis.end()), lis.end());
+}
+
+void removeDuplicate(std::deque<int>& deq) {
+    std::sort(deq.begin(), deq.end());
+    deq.erase(std::unique(deq.begin(), deq.end()), deq.end());
+}
+
+void TranformToMap(std::list<std::string>& lis, std::deque<int>& deq, std::map<int, std::string>& map) {
+    if (lis.size() != deq.size()) {
+        std::cerr << "Containers must be of the same size";
+    } else {
```

Comment:
Normally I will not put this comment here, but `Lukasz K/Nanautzin` wrote that I don't leave a comment ;p so prefer

```
if (...) {
   return;
}

return
```

Instead of if - else